src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -369,40 +433,73 @@ def getJobStatus(self, jobIDList):
             job, _, jobID = condorIDAndPathToResultFromJobRef(jobRef)
             condorIDs[job] = jobID
 
+        self.tokenFile = None
+
         qList = []
         for _condorIDs in breakListIntoChunks(condorIDs.values(), 100):
             # This will return a list of 1245.75 3
-            status, stdout_q = subprocess.getstatusoutput(
-                "condor_q {} {} -af:j JobStatus ".format(self.remoteScheddOptions, " ".join(_condorIDs))
-            )
+            cmd = ["condor_q"]
+            cmd.extend(self.remoteScheddOptions.strip().split(" "))
+            cmd.extend(_condorIDs)
+            cmd.extend(["-af:j", "JobStatus"])
+            result = self._executeCondorCommand(cmd, keepTokenFile=True)
+            if not result["OK"]:
+                self.tokenFile = None
+                return S_ERROR("condor_q failed completely: %s" % result["Message"])
+            status, stdout, stderr = result["Value"]
             if status != 0:
-                return S_ERROR(stdout_q)
-            _qList = stdout_q.strip().split("\n")
+                self.tokenFile = None
+                return S_ERROR(stdout + stderr)
+            _qList = stdout.strip().split("\n")
             qList.extend(_qList)
 
             # FIXME: condor_history does only support j for autoformat from 8.5.3,
             # format adds whitespace for each field This will return a list of 1245 75 3
-            # needs to cocatenate the first two with a dot
+            # needs to concatenate the first two with a dot
             condorHistCall = "condor_history {} {} -af ClusterId ProcId JobStatus".format(
```

Comment:
```suggestion
            condorHistCall = f"condor_history {result['Message']} {" ".join(_condorIDs)} -af ClusterId ProcId JobStatus".
```

Reply:
Flynt does not touch these lines. It is not almighty !